### PR TITLE
Use the pager in more places

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/standard-options.xml [hunk 1]
```diff
@@ -29,6 +31,16 @@
     </listitem>
   </varlistentry>
 
+  <varlistentry id='no-pager-255'>
```
Comment (YHNdnzj) [inline man/standard-options.xml:34]: Hmm, I don't really like having "versioned" options in `standard-options`, especially when the option already exists, but just not hooked up with some tools before.

**Review by YHNdnzj — APPROVED** [review]

File: man/standard-options.xml (hunk 1 above)
Comment (keszybz) [reply man/standard-options.xml:34]: Do you have a better idea? For this particular tool, the option appears in version 255. The fact that the same option appeared in 214 in some other tool is not really relevant. By using standard-options, we minimize the number of copies to at most one per version. Also, all those text are together in the same file, so if we decide to change the wording one day, we can do this easily. If we had something better than docbook xml, that could do composition or macros or something, we probably could avoid such duplication. But this is what we have right now.

Comment (YHNdnzj) [reply man/standard-options.xml:34]: OK. This is convincing.

File: src/path/path.c [hunk 2]
```diff
@@ -12,10 +12,12 @@
 #include "log.h"
 #include "macro.h"
 #include "main-func.h"
+#include "pager.h"
 #include "pretty-print.h"
 #include "string-util.h"
 
 static const char *arg_suffix = NULL;
+PagerFlags arg_pager_flags = 0;
```
Comment (poettering) [inline src/path/path.c:20]: should be static

**Pull request merged**